public/assets/js/chart.js

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
This is hard-coded  data to verify the proof of concept.  Onboarding more dynamic content later once backend built out.